### PR TITLE
PCR-802 Add runbook attribute to the base class

### PR DESCRIPTION
**Pull request by nickyinhu**
So that a runbook link can be passed in when creating the class and
can be displayed.

### CODE REVIEW

File: lib/HealthCheck/Diagnostic.pm [hunk 1]
```diff
@@ -275,7 +289,7 @@ sub check {
 Validates, pre-formats, and returns the C<result> so that it is easily
 usable by HealthCheck.
 
-The attributes C<id>, C<label>, and C<tags>
+The attributes C<id>, C<label>, C<runbook> and C<tags>
```
Comment (afresh1) [inline lib/HealthCheck/Diagnostic.pm]: oxford comma, please

File: lib/HealthCheck.pm [hunk 2]
```diff
@@ -34,20 +34,22 @@ Hash::Util::FieldHash::fieldhash my %registered_checks;
     }
 
     my $checker = HealthCheck->new(
-        id     => 'main_checker',
-        label  => 'Main Health Check',
-        tags   => [qw( fast cheap )],
-        checks => [
+        id      => 'main_checker',
+        label   => 'Main Health Check',
+        runbook => 'https://runbook.url',
```
Comment (afresh1) [inline lib/HealthCheck.pm]: Please use [a reserved TLD for the examples](https://datatracker.ietf.org/doc/html/rfc2606#section-2) or one that "belongs" to us like https://grantstreetgroup.github.io/HealthCheck.html.

File: t/HealthCheck.t [hunk 3]
```diff
@@ -335,24 +339,28 @@ my $nl = Carp->VERSION >= 1.25 ? ".\n" : "\n";
     is $c->check, {
         'id'      => 'main',
         'status'  => 'CRITICAL',
+        'runbook' => 'https://runbook-main.url',
```
Comment (afresh1) [inline t/HealthCheck.t]: Use a reserved TLD (like .test) in tests.

**Review by afresh1 — COMMENTED** [review]
Do we want to add any validation of the "runbook" that's returned by a check?  I would expect it to be a string and maybe even a uri.  But perhaps we can leave it open and allowing a reference to a list of runbooks or a map that includes the url as well as a title and type or something.  

Other than that and the minor comments about the correct tld, this looks great.  Thanks!

File: lib/HealthCheck/Diagnostic.pm (hunk 1 above)
Comment (nickyinhu) [reply lib/HealthCheck/Diagnostic.pm]: Added, thanks

File: lib/HealthCheck.pm (hunk 2 above)
Comment (nickyinhu) [reply lib/HealthCheck.pm]: Fixed

File: t/HealthCheck.t (hunk 3 above)
Comment (nickyinhu) [reply t/HealthCheck.t]: Fixed

**Review by afresh1 — APPROVED** [review]

**Pull request merged**